src/main/java/net/opentsdb/tsd/KafkaRpcPluginConfig.java

```diff
@@ -37,43 +37,29 @@
   // KAFKA
   public static final String AUTO_COMMIT_INTERVAL_MS = 
       "auto.commit.interval.ms";
-  public static final String AUTO_COMMIT_ENABLE = "auto.commit.enable";
+  public static final String AUTO_COMMIT_ENABLE = "enable.auto.commit";
   public static final String AUTO_OFFSET_RESET = "auto.offset.reset";
-  public static final String REBALANCE_BACKOFF_MS = "rebalance.backoff.ms";
-  public static final String REBALANCE_RETRIES = "rebalance.retries.max";
-  public static final String ZOOKEEPER_CONNECTION_TIMEOUT_MS =
-      "zookeeper.connection.timeout.ms";
-  public static final String ZOOKEEPER_SESSION_TIMEOUT_MS = 
-      "zookeeper.session.timeout.ms";
+  public static final String REBALANCE_BACKOFF_MS = "rebalance.timeout.ms";
   
   // KAFKA DEFAULTS
   public static final int AUTO_COMMIT_INTERVAL_DEFAULT = 5000;
-  public static final String AUTO_COMMIT_ENABLE_DEFAULT = "true";
-  public static final String AUTO_OFFSET_RESET_DEFAULT = "smallest";
+  public static final String AUTO_COMMIT_ENABLE_DEFAULT = "false";
+  public static final String AUTO_OFFSET_RESET_DEFAULT = "latest";
```

Comment:
To keep the changeset as small as possible and avoid surprises for users, I'd lean to keeping the old defaults in place unless there is a particular reason why have these two defaults changed?

Reply:
Auto-commit is off-by-default because this code does manual committing after we insert a batch of records.

For offset_reset, kafka changed the value names. "latest" is the new name for "smallest".